packages/synapse-react-client/stories/IconSvg.stories.tsx

```diff
@@ -8,7 +8,7 @@ const meta = {
   render: args => {
     return (
       <div style={{ display: 'flex', flexWrap: 'wrap', flexDirection: 'row' }}>
-        {IconStrings.sort().map(icon => (
+        {IconStrings.toSorted().map(icon => (
```

Comment:
Small tweak in this story, `toSorted` makes a copy instead of mutating in-place. Safer, but probably doesn't make a difference.

Reply:
Thanks Nick. I just looked into toSorted too.